### DISCUSSION TITLE
garbage SOFTWARE

### DISCUSSION THREAD

**Discussion by castaloco** [Chia-Network/chia-blockchain#4893] [General]
this shit keeps desync there no button no sync manually, it can work properly for days, but after desync nothing, make a fucking button or a built in script to reconnect itself it pisses my off losing my plots.

**Comment by chiabug** [comment]
I feel your pain.  I suggest making certain that your router has port 8444 open and forwarded to the IP of your machine. This seems to solve many sync problems.

**Comment by CCarroz** [comment]
Super productive feedback on free software.  Look at your logs and at least try to offer some level of productive feedback. The issue could be in the software or it could be 100% on your end.  Maybe someone in your house blocked port 8444 just to mess with you.
